openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -62,7 +62,7 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://media.sailthru.com/595/1k1/8/o/599f355101b3f.png" width="70"
+                                    src="https://edx-cdn.org/v3/prod/logo.png" width="70"
```

Comment:
Also, I thought you determined that the `SVG` variant of the image is required for the email templates, no?

Reply:
Ah, other way around, We need non `SVG` image for the email templates.

Reply:
Yes, of course! Thanks for the correction!  #head-palm